libraries/chain/webassembly/privileged.cpp

```diff
@@ -277,4 +278,4 @@ namespace eosio { namespace chain { namespace webassembly {
 }}} // ns eosio::chain::webassembly
 
 FC_REFLECT(eosio::chain::webassembly::abi_finalizer_authority, (description)(fweight)(public_key_g1_affine_le));
```

Comment:
Rename all these to match names in CDT

Reply:
In CDT hotstuff_integration branch, those names have not changed. They are matched.

Reply:
Can you get a PR up to CDT to fix that?

Reply:
Will do.

Reply:
I would like to get this first pass in, and clean up names in future CDT, Leap, and Reference-Contracts PRs.

Reply:
Will be done by CDT https://github.com/AntelopeIO/cdt/issues/257